x-pack/plugins/cases/public/components/case_view/case_view_page.test.tsx

```diff
@@ -575,4 +578,91 @@ describe('CaseViewPage', () => {
       });
     });
   });
+
+  describe('Tabs', () => {
+    let appMockRender: AppMockRenderer;
+    beforeEach(() => {
+      appMockRender = createAppMockRenderer();
+    });
+    it('renders tabs correctly', async () => {
+      const result = appMockRender.render(<CaseViewPage {...caseProps} />);
+      await act(async () => {
+        expect(result.getByTestId('case-view-tab-title-alerts')).toBeTruthy();
+        expect(result.getByTestId('case-view-tab-title-activity')).toBeTruthy();
+      });
+    });
+
+    it('renders the activity tab by default', async () => {
```

Comment:
What is the difference between this test and the previous one?

Reply:
one tests that the tabs are rendered, the other that the content is rendered.

Reply:
Oh, I see. Thanks!